include/vctr/Expressions/Readme.md

```diff
@@ -183,7 +183,7 @@ the expression:
 - SSE: Implement `SSERegister<value_type> getSSE (size_t i) const` for suitable types. Make sure to require at
   least `archX64 && has::getSSE<SrcType>`
 - Neon: Implement `NeonRegister<value_type> getNeon (size_t i) const` for suitable types. Make sure to require at
-  least `archARM && has::getAVX<SrcType>`
+  least `archARM && has::getNeon<SrcType>`
```

Comment:
yuk…

Reply:
Fortunately only in the documentation ;)